### PR TITLE
Bunch of modifications regarding LoginPage, Header and creation of loginHeader

### PR DESCRIPTION
**Pull request by Amir8Abdullah**
1. In .github/workflows/main.yml: change the branch condition from -master to -main.
2. The login page is now fully functional: desgined the full backend handling of the login page.
3. Header: Removed uncesserary part in the header.
4. Login Header: Created a new header for users who are logged in. The correspondant CSS file has been modified as well.

### CODE REVIEW

**Comment by Razvan-Alexandru** [comment]
don't use the same pull request for different tasks plz

**Review by Razvan-Alexandru — CHANGES_REQUESTED** [review]
Do not use the same pull request for different tasks

**Comment by Amir8Abdullah** [comment]
It has been added to epic/loginPage.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: CI
 on:
   push:
     branches:
-      - master
```
Comment (Razvan-Alexandru) [inline .github/workflows/main.yml:6]: do not change here

**Review by Razvan-Alexandru — DISMISSED** [review]

**Review by Razvan-Alexandru — CHANGES_REQUESTED** [review]
do not change main.yml in this branch

**Review by Razvan-Alexandru — APPROVED** [review]

**Pull request merged**